lib/AST/ASTContext.cpp

```diff
@@ -3951,8 +3951,17 @@ bool ASTContext::isStandardLibraryTypeBridgedInFoundation(
           nominal == getStringDecl() ||
           nominal == getErrorDecl() ||
           nominal == getAnyHashableDecl() ||
-          // Weird one-off case where CGFloat is bridged to NSNumber.
-          nominal->getName() == Id_CGFloat);
+          // Foundation is a dependency of CoreGraphics, but
```

Comment:
What kind of an impact do you have in mind? Generating a bunch of conformances bloats the binary and metadata. I guess that could be measured. I don't think the optimizer makes use of knowledge of the absence of a conformance. I don't think we could make that assumption for a public type.